main.js

```diff
@@ -0,0 +1,238 @@
+
+let start = document.getElementById('start');
+let playAgain = document.getElementById('playAgain');
+let status = document.getElementById('status');
+let cell0 = document.getElementById('cell-0');
+let cell1 = document.getElementById('cell-1');
+let cell2 = document.getElementById('cell-2');
+let cell3 = document.getElementById('cell-3');
+let cell4 = document.getElementById('cell-4');
+let cell5 = document.getElementById('cell-5');
+let cell6 = document.getElementById('cell-6');
+let cell7 = document.getElementById('cell-7');
+let cell8 = document.getElementById('cell-8');
+let row1 = document.getElementById('row-1');
+let row2 = document.getElementById('row-2');
+let row3 = document.getElementById('row-3');
+row1.hidden = true;
+row2.hidden = true;
+row3.hidden = true;
+playAgain.hidden = true;
+playAgain.disabled = true;
+// let playerX = document.getElementById('playerX');
+// let playerO = document.getElementById('playerO');
+// let nameX = document.getElementById('nameX');
+// let nameO = document.getElementById('nameO');
+
+let curPlayer = 'X';
+
+// functions
+
+// function playerNames() {
+//     playerX = firstPlayer;
+//     pl.innerHTML = 'What is the name for X player?';
+//     return status;
+// }
+
+
+function startGame() {
+    start.disabled = true;
+    start.hidden = true;
+    row1.hidden = false;
+    row2.hidden = false;
+    row3.hidden = false;
+    console.log('this is the start of the game');
+    cell0.disabled = false;
+    status.innerHTML = "Player " + curPlayer + "'s turn";
+    return status;
+};
+
+function startAgain() {
+    location.reload();
+    start.disabled = false;
+    start.hidden = false;
+    playAgain.disabled = false;
+    playAgain.hidden = false;
+    return startGame();
+}
+
+function currentPlayer() {
+    status.innerHTML = "Player " + curPlayer + "'s turn";
+    return status;
+};
+
+function curPlay0() {
+    console.log('This is the current player ' + curPlayer)
+    if (curPlayer === 'X') {
+        cell0.innerHTML = 'X';
+        let html = cell0.innerHTML;
+        console.log('This is what is in cell 0 ' + html);
+        curPlayer = 'O';
+        cell0.removeEventListener('click', curPlay0, true);
+        return whoWon('X')
+    } else
+        cell0.innerHTML = 'O';
+    curPlayer = 'X';
+    cell0.removeEventListener('click', curPlay0, true);
+    return whoWon('O');
+};
+
+function curPlay1() {
```

Comment:
All of these `curPlayX` functions are *very* similar and should probably be a single function with a `cellNumber` parameter.